packages/manager/apps/pci-instances/src/pages/instances/delete/DeleteInstance.page.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { useNavigate, useParams, useSearchParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useNotifications } from '@ovh-ux/manager-react-components';
+import { useDeleteInstance } from '@/data/hooks/instance/useDeleteInstance';
+import { updateDeletedInstanceStatus } from '@/data/hooks/instance/useInstances';
+import queryClient from '@/queryClient';
+import { DeleteModal } from './modal/DeleteModal.component';
+
+const DeleteInstance: FC = () => {
+  const { t } = useTranslation(['delete', 'common']);
+  const navigate = useNavigate();
+  const { projectId } = useParams() as { projectId: string };
+  const [searchParams] = useSearchParams();
+  const instanceId = searchParams.get('instanceId');
+  const instanceName = searchParams.get('instanceName');
+  const { addError, addSuccess } = useNotifications();
+
+  const handleModalClose = () => {
```

Comment:
action methods should be wrapped in useCallback

Reply:
Well, I feel that it's not necessary here as there is not a heavy computation that needs a memoization.